src/selectors/entities/preferences.js

```diff
@@ -138,27 +138,19 @@ export const getTheme = createShallowSelector(
         // At this point, the theme should be a plain object
 
         // If this is a system theme, find it in case the user's theme is missing any fields
-        let baseTheme = Preferences.THEMES.default;
-        if (theme.type) {
-            for (const key of Object.keys(Preferences.THEMES)) {
-                if (Preferences.THEMES[key].type === theme.type) {
-                    baseTheme = Preferences.THEMES[key];
-                    break;
-                }
-            }
+        const baseTheme = Preferences.THEMES.default;
+        if (theme.type && theme.type !== 'custom') {
+            return baseTheme;
```

Comment:
This should use the matching system theme like it was doing before, but it should return the whole theme instead of using that to cherry-pick missing fields